### ISSUE TITLE
Graphics issue on tetris evolution

### ISSUE THREAD

**Issue by fireclouu** [nikita36078/J2ME-Loader#100]
![screenshot_2018-02-07-10-02-43](https://user-images.githubusercontent.com/22563129/35894530-1acbe51e-0bee-11e8-9596-8dc68d3f03d0.png)
![screenshot_2018-02-07-10-02-47](https://user-images.githubusercontent.com/22563129/35894539-27f2a200-0bee-11e8-8aef-7db534b34b42.png)

This is **Tetris Revolution** appear on the emulator.  Screen resolution is `240x320` , without enabling any hacks `Immediate processing mode` and `Draw buffer clearing`.

The following is the Main menu screen that shows graphical glitches before and after using hacks:

> Without hacks:
![screenshot_2018-02-07-10-02-13](https://user-images.githubusercontent.com/22563129/35894771-5f3ab8c8-0bef-11e8-807e-5fac9a404655.png)

> With hacks:
![screenshot_2018-02-07-09-51-30](https://user-images.githubusercontent.com/22563129/35894786-711bd4d2-0bef-11e8-819d-4bbd966920b6.png)

**Comment by oatmeal01** [comment]
seems that the non-touchscreen version has a bug, downloaded  240x320 s40v5 version and experienced the same issue. If you want to play the game smoothly try downloading the touchscreen version.

**Comment by jasper1412** [comment]
@oatmeal01 thanks for suggesting that, but I really am not that interested on the game, its just that it needs to be reported so that the emulation achieve more accurate result. I'm really waiting for the fix on #93 , old school Meboy rocks! even though there are high-quality gb/c emulators available on market :)

**Comment by amarcioz** [comment]
I want to play this game.

**Comment by jasper1412** [comment]
1.2.7.9-open makes the game lags even more than usual even though `Filter` is disabled.

**Comment by jasper1412** [comment]
Before I forgot, it was version 1.13.73

**Comment by AdityaMH** [comment]
I think since adding touch input in previous version, much game with touchscreen support (mostly gameloft) very lag and hard to response. But for un-touchscreen game can run perfectly without any problem.

**Comment by jasper1412** [comment]
tried it on 1.4.9-play, still has those bugs. I tried comparing 2 versions of it , non touchscreen 1.11.71 and touchscreen 1.13.6, with the same 240x320 screensize. 1.11.71 graphics did executed well, while the superior one with so much effects and transitions suffers on like on the screenshots above. I had another device now with adreno gpu and still causes glitches.

I tried looking for answers why this is happening although these errors were self-explanatory (well because of added/updated functionality of j2me instructions). Tried loading these applet on pspkvm and runs fine so now I suspect if some instructions aren't fully implemented.
